docs/collaboration/governance.md

```diff
@@ -280,35 +280,35 @@ OpenStand:
 
 ### B. Standard decision making process overview
 
-The foundation of BIDS decision making is listening to all members of
-the BIDS Community and striving to achieve consensus on each level of
-the BIDS standard process.
+The foundation of BIDS decision making is listening to all members of the BIDS Community
+and striving to achieve consensus on each level of the BIDS standard process.
 
 The criteria for forming a new BEP Working Group:
 
 -   Statement of intent with defined scope, deliverables, and use cases
```

Comment:
This could be clearer -- maybe just add "... defined within a proposal document [BIDS Extension Proposal (BEP)] "
because otherwise it's a bit  unclear for "forming" a WG - chicken-and-egg does starting a googledoc or drafting a separate  "statement of intent" come first ?